http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcBaseTest.java

```diff
@@ -85,6 +86,7 @@ public class OidcBaseTest extends AbstractBaseHttpTest {
     public static final String CLIENT_SECRET = "longerclientsecretthatisstleast256bitslong";
     public static KeycloakContainer KEYCLOAK_CONTAINER;
     public static final String TEST_REALM = "WildFly";
+    public static final String TEST_REFRESH_REALM = "WildFlyRefresh";
```

Comment:
Just one last small comment - if this is unused, it can be removed.

Reply:
removed